install.md

```diff
@@ -117,7 +117,9 @@ sudo yum install -y \
   ostree-devel \
   pkgconfig \
   runc \
-  containers-common
+  containers-common \
+  bcc-devel \
+  kernel-headers
```

Comment:
We'll need similar changes here too me thinks:  https://github.com/containers/libpod/blob/master/contrib/podmanimage/upstream/Dockerfile#L20